packages/blackarch-helper/blackarch-helper

```diff
@@ -0,0 +1,516 @@
+#!/usr/bin/env bash
```

Comment:
Please don't copy all source files locally, deploy them from git. See this template:  https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-generic-git

Reply:
PKGBUILD is better, and you removed SRCINFO, but all the local files are still here, please remove LICENSE, _blackarch-helper, etc. anything but the PKGBUILD. All other files will be fetched from git at build time